src/pages/ParkJuHee/Login/Login.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+import './Login.scss';
+import { Link, useNavigate } from 'react-router-dom';
+
+const Login = () => {
+  const navigate = useNavigate();
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [isValid, setIsValid] = useState(false);
+
+  const handleEmail = (e) => {
+    setEmail(e.target.value);
+    setIsValid(isValidEmail(e.target.value));
+  };
+  const handlePassword = (e) => {
+    setPassword(e.target.value);
+  };
+  const isValidEmail = (email) => {
+    const emailPattern = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
+    return emailPattern.test(email);
+  };
+
+  const handleLogin = (body) => {
+    fetch('http://localhost:8000/user/signup', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+      },
+      body: JSON.stringify(body),
+    })
+      .then((res) => res.json())
+      .then((data) => {
+        if (data.message === 'LOGIN SUCCESS') {
+          console.log('success :', data);
+          navigate('/main');
+          localStorage.setItem('token', data.accessToken);
+        } else {
+          console.log('failed :', data);
+        }
+      })
+      .catch((err) => console.log(err));
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    let body = {
+      email: email,
+      password: password,
+    };
```

Comment:
객체를 구성하는 key와,  value로 지정된 변수명이 같기때문에
아래와 같이  수정할  수 있습니다
```suggestion
    let body = {
      email,
      password,
    };
```